test/gtest/CMakeLists.txt

```diff
@@ -168,7 +168,7 @@ else()
     install(TARGETS miopen_gtest
         PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
         DESTINATION ${CMAKE_INSTALL_BINDIR}
-        COMPONENT client)
```

Comment:
@xinlipn mmm ... this typo should not have happened. Thanks for catching this, let me know the impacts and if we need to move to other branches for this fix.

Reply:
> @xinlipn mmm ... this typo should not have happened. Thanks for catching this, let me know the impacts and if we need to move to other branches for this fix.

@junliume , No problem. QA team may this this change for MIOpen Test Packages, may need your help to promote to mainline/amd-master. Thanks